session test panicking

```
panic: runtime error: slice bounds out of range

goroutine 35 [running]:
github.com/lucas-clemente/quic-go.(*packetPacker).composeNextPacket(0xc4241f5960, 0x0, 0x4fd, 0x0, 0x0, 0x0, 0x0, 0xc424624b00)
	/root/src/go/src/github.com/lucas-clemente/quic-go/packet_packer.go:231 +0x169
github.com/lucas-clemente/quic-go.(*packetPacker).packPacket(0xc4241f5960, 0x0, 0x1, 0x0, 0xc42400aca0, 0x1, 0x1)
	/root/src/go/src/github.com/lucas-clemente/quic-go/packet_packer.go:150 +0xbd5
github.com/lucas-clemente/quic-go.(*packetPacker).PackPacket(0xc4241f5960, 0x0, 0xc42400aca0, 0x1, 0x1, 0x1, 0x0, 0x1, 0x0)
	/root/src/go/src/github.com/lucas-clemente/quic-go/packet_packer.go:75 +0x118
github.com/lucas-clemente/quic-go.(*session).sendPacket(0xc423ff7ba0, 0x0, 0x0)
	/root/src/go/src/github.com/lucas-clemente/quic-go/session.go:614 +0xab1
github.com/lucas-clemente/quic-go.(*session).run(0xc423ff7ba0, 0x0, 0x0)
	/root/src/go/src/github.com/lucas-clemente/quic-go/session.go:248 +0x273
created by github.com/lucas-clemente/quic-go.glob..func11.18.3
	/root/src/go/src/github.com/lucas-clemente/quic-go/session_test.go:1081 +0xd1
```

--- a/session_test.go
+++ b/session_test.go
@@ -103,7 +103,11 @@ type mockReceivedPacketHandler struct {
 	nextAckFrame *frames.AckFrame
 }
 
-func (m *mockReceivedPacketHandler) GetAckFrame() *frames.AckFrame { return m.nextAckFrame }
+func (m *mockReceivedPacketHandler) GetAckFrame() *frames.AckFrame {
+	f := m.nextAckFrame
+	m.nextAckFrame = nil
+	return f
+}
 func (m *mockReceivedPacketHandler) ReceivedPacket(packetNumber protocol.PacketNumber, shouldInstigateAck bool) error {
 	panic("not implemented")
 }

